Automattic/wp-calypso
# InlineHelp: Add admin help sections data

#### Changes proposed in this Pull Request

This PR adds the help admin sections data to the inline help component. It's part of the process to restore the help admin section for the InlineHelp and HelpSearch cards.
The way to access the data is through of a state selector, named `getAdminHelpResults()`.

#### Testing instructions

* Take a look at the admin items defined in the `admin-sections.js` file. Check typos, descriptions, synonyms, etc.
* Run unit tests

```
> yarn test-client client/state/inline-help/test/selectors.js
```

```
 PASS  client/state/inline-help/test/reducer.js
 PASS  client/state/inline-help/test/selectors.js

Test Suites: 2 passed, 2 total
Tests:       32 passed, 32 total
Snapshots:   0 total
Time:        3.431 s, estimated 4 s
Ran all test suites matching /client\/state\/inline-help\/test/i.

Watch Usage: Press w to show more.
✨  Done in 2249.24s.
```

```
> yarn test-client client/blocks/inline-help/test/admin-sections.js
```

```
 PASS  client/blocks/inline-help/test/admin-setions.js

Test Suites: 1 passed, 1 total
Tests:       5 passed, 5 total
Snapshots:   0 total
Time:        3.957 s
Ran all test suites matching /client\/blocks\/inline-help\/test\/admin-setions.js/i.
✨  Done in 8.54s.
```

All tests should pass.

Fixes #43464
